data-raw/R2R_add_upper_san_joaquin/add_upper_san_joaquin_river_habitat.Rmd

```diff
@@ -151,11 +151,13 @@ yuba_suit <- suitability |>
 
 ```
 
+## Background
 
+This document describes the process of adding instream juvenile and fry rearing, spawning, and floodplain rearing to the Upper San Joaquin reach of the [SpringRunDSM](https://github.com/Reorienting-to-Recovery/springRunDSM) (from Merced River confluence to Friant Dam). The Lower San Joaquin reach is 45.7 miles long while the Upper San Joaquin Reach is 153 miles in length. There is only one San Joaquin reach in the 2018/2019 BiOp CalSim and therefore habitat data for Lower and Upper San Joaquin River use flow data from CalSim node C630. 
 
 ## Instream Rearing Habitat
 
-The 1D Hec Ras model was run for for flows 100 through 4500 CFS at all nodes of the bypasses (Chowchilla, Eastside, and Mariposa) and reaches 1, 1A, 2A, and 5. A suitability based on depth and velocity was applied all nodes and flows to get a suitable area for in-channel and floodplain rearing for the Upper San Joaquin River between Merced River and Friant Dam. 
+The 1D Hec Ras model was run for for flows 100 through 4500 CFS at all nodes of the bypasses (Chowchilla, Eastside, and Mariposa) and reaches 1, 1A, 2A, and 5. A suitability criteria based on depth and velocity was applied to all nodes and flows to get a suitable area for in-channel and floodplain rearing for the Upper San Joaquin River between Merced River and Friant Dam. 
 
 Suitability criteria was applied to the outputs of the Hec Ras model for suitable depth and velocity for instream juvenile and fry rearing. The Yuba River depth and velocity criteria was used as a proxy for the Upper San Joaquin River using data provided by Mark Gard from the California Department of Fish and Wildlife. 
```

Comment:
What is "The Yuba River" depth and velocity criteria. Please link and describe the document